### PR TITLE
chore(customers): CUST-1945 update address schema for create customers

### PR DESCRIPTION
**Pull request by mjhmatt**
Updating schema for customers endpoint with address form fields, also updating response body of customers to include form fields

API request: 
image

Documentation Request:
image

Documentation Response:
image

### CODE REVIEW

File: reference/customers.v3.yml [hunk 1]
```diff
@@ -3292,10 +3295,12 @@ components:
...
         form_fields:
-          description: Array of form fields. Controlled by `formfields` parameter.
-          type: array
-          items:
-            $ref: '#/components/schemas/formFieldValue_Full'
+                description: Array of form fields. Controlled by `formfields` parameter.
+                type: array
+                items:
+                  allOf:
+                    - $ref: "#/components/schemas/formFieldValue_CustomerAddress"
```
Comment (bc-akshatkaushik) [inline reference/customers.v3.yml:3302]:
```suggestion
                    - $ref: "#/components/schemas/formFieldValue"
```
https://github.com/bigcommerce/api-specs/pull/699/files#diff-e32e9f10280459c2f2dd1cd85354d06cc59a229a38846daf423e065b81a5b25fR3609

File: reference/customers.v3.yml [hunk 2]
```diff
@@ -3684,6 +3689,32 @@ components:
...
+              format: double
+              example: 12.345
+            - type: array
+              example:
+                - red
+                - green
+                - black
+              items:
+                type: string
+      x-internal: false
```
Comment (bc-akshatkaushik) [inline reference/customers.v3.yml:3717]:
```suggestion
```

**Comment by bc-keith** [comment]
@slsriehl -- This looks like it rolls up to https://jira.bigcommerce.com/browse/CSS-578 as CUST-1945 is a sub-task of CUST-1941, which is in the PROJECT-4463 epic.

**Comment by slsriehl** [comment]
@bc-keith is there a DEVDOCS ticket associated? If so, please link. If not, please create.

**Comment by bc-keith** [comment]
@slsriehl, yes, it is either going to be https://jira.bigcommerce.com/browse/DEVDOCS-3435 or https://jira.bigcommerce.com/browse/DEVDOCS-3539. Both are linked to the CSS ticket.

**Comment by slsriehl** [comment]
@bc-tgomez since this is now a ticket branch, go ahead and work it (as you're on customers, no?). When you're happy with it, merge this PR. It'll be ready to go to staging and then master as @bc-keith gives the all clear from the coordinated launch POV.

**Pull request merged**